packages/react/src/components/SuiteHeader/SuiteHeader.jsx

```diff
@@ -389,16 +392,21 @@ const SuiteHeader = ({
                 />
               </span>
             ),
-            onClick: async () => {
+            onClick: async (e) => {
               let href = routes.admin;
               let routeType = ROUTE_TYPES.ADMIN;
+              const newWindow = shouldOpenInNewWindow(e);
               if (isAdminView) {
                 href = navigatorRoute;
                 routeType = ROUTE_TYPES.NAVIGATOR;
               }
               const result = await onRouteChange(routeType, href);
               if (result) {
-                window.location.href = href;
+                if (newWindow) {
+                  window.open(href, '_blank', 'noopener noreferrer');
```

Comment:
Nice. I think adding the noopener and noreferrer here makes sense since they are becoming the default when using normal _blank links